lib/data/network/recovery_words/recovery_words_api.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:fluffychat/data/model/recovery_words_json.dart';
+import 'package:fluffychat/data/network/dio_client.dart';
+import 'package:fluffychat/data/network/tom_endpoint.dart';
+import 'package:fluffychat/di/global/get_it_initializer.dart';
+import 'package:fluffychat/di/global/network_di.dart';
+
+class RecoveryWordsAPI {
+  final DioClient _client = getIt.get<DioClient>(instanceName: NetworkDI.tomDioClientName);
```

Comment:
for shorter name `final _client = getIt.get<DioClient>(instanceName: NetworkDI.tomDioClientName);`